File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Data/Domain/PEC/PECMailBoxConfiguration.vb
<Serializable()> _
Public Class PECMailBoxConfiguration
    Inherits DomainObject(Of Integer)

    Public Overridable Property Name As String

    Public Overridable Property MaxReadForSession As Integer

    Public Overridable Property MaxSendForSession As Integer

    Public Overridable Property UnzipAttachments As Boolean

    Public Overridable Property MarkAsRead As Boolean

    Public Overridable Property MoveToFolder As String

    Public Overridable Property MoveErrorToFolder As String

    Public Overridable Property InboxFolder As String

    Public Overridable Property UploadSent As Boolean

    Public Overridable Property FolderSent As String

    Public Overridable Property ImapSearchFlag As ImapFlag

    Public Overridable Property ImapStartDate As Date?

    Public Overridable Property ImapEndDate As Date?

    Public Overridable Property NoSubjectDefaultText As String

    Public Overridable Property DeleteMailFromServer As Boolean?

    Public Overridable Property MaxReceiveByteSize As Long

    Public Overridable Property MaxSendByteSize As Long

End Class

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Entity/PECMails/PECMailBoxConfiguration.cs
using System;

namespace VecompSoftware.DocSuiteWeb.Entity.PECMails
{
    public class PECMailBoxConfiguration : DSWBaseEntity
    {
        #region [ Constructor ]
        public PECMailBoxConfiguration() : this(Guid.NewGuid()) { }
        public PECMailBoxConfiguration(Guid uniqueId)
        : base(uniqueId)
        { }
        #endregion

        #region [ Properties ]
        public string Name { get; set; }
        public int? MaxReadForSession { get; set; }
        public int? MaxSendForSession { get; set; }
        public bool? UnzipAttachments { get; set; }
        public int? SslPort { get; set; }
        public bool? ImapEnabled { get; set; }
        public bool? UseImapSsl { get; set; }
        public int? ImapPort { get; set; }
        public bool? MarkAsRead { get; set; }
        public string MoveToFolder { get; set; }
        public string MoveErrorToFolder { get; set; }
        public string InboxFolder { get; set; }
        public bool? UploadSent { get; set; }
        public string FolderSent { get; set; }
        public int? ImapSearchFlag { get; set; }
        public DateTime? ImapStartDate { get; set; }
        public DateTime? ImapEndDate { get; set; }
        public string NoSubjectDefaultText { get; set; }
        public bool? DeleteMailFromServer { get; set; }
        public short? ReceiveDaysCap { get; set; }
        public long? MaxReceiveByteSize { get; set; }
        public long? MaxSendByteSize { get; set; }
        #endregion

        #region [ Navigation Properties ]

        #endregion
    }
}
